v1/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "distributed-press-api",
+  "version": "0.1.0",
+  "description": "API for the Distributed Press",
+  "main": "build/index.js",
+  "type": "module",
+  "scripts": {
+    "clean": "rm -rf build",
+    "build": "npm run clean && tsc",
+    "lint": "tsc --noEmit && ts-standard --fix",
+    "dev": "ts-node-esm index.ts | pino-pretty -c -t",
+    "watch": "nodemon --watch './**/*.ts' --exec 'node --experimental-specifier-resolution=node --loader ts-node/esm' index.ts | pino-pretty -c -t",
+    "start": "node build/index.js",
+    "test": "ava"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyphacoop/api.distributed.press.git"
+  },
+  "keywords": [
+    "publish",
+    "dweb",
+    "hypercore",
+    "dat",
+    "ipfs",
+    "web monetization",
+    "ethereum"
+  ],
+  "author": "Distributed Press",
+  "license": "AGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/hyphacoop/api.distributed.press/issues"
+  },
+  "homepage": "https://distributed.press",
+  "dependencies": {
+    "@fastify/multipart": "^7.3.0",
+    "@fastify/swagger": "^8.2.0",
+    "@fastify/swagger-ui": "^1.3.0",
+    "@fastify/type-provider-typebox": "^2.3.0",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "fastify": "^4.10.2",
+    "fastify-metrics": "^10.0.0",
+    "fs": "0.0.1-security",
+    "nodemon": "^2.0.20",
+    "prom-client": "^14.1.0",
+    "ts-node": "^10.9.1"
```

Comment:
Should this be in dev dependencies along with nodemon? Or is there a benefit of having it be in the main dependencies?

Reply:
I think these are needed to run it if we rely on ts-node? Correct me if I'm wrong haha

Reply:
It doesn't matter ultimately since there isn't anything consuming this as a dependency, but IMO ts-node would make more sense in the dev dependencies with any projects using this as a dependency needing to install it for themsevles.

Generally any utilities that are used in `scripts` but aren't explicitly imported in the code (non-test code that is) should be in dev dependencies.